pkg/controller/pxc/controller.go

```diff
@@ -199,35 +195,36 @@ func (r *ReconcilePerconaXtraDBCluster) Reconcile(request reconcile.Request) (re
 		return reconcile.Result{}, err
 	}
 
+	reqLogger := log.WithValues("cluster name", o.Name, "namespace", o.Namespace)
```

Comment:
I think It's better to have global logger and not to pass it as a argument.

Reply:
we can't have global logger because requests could come in parallel

Reply:
Maybe we can have logger store then?

Reply:
6af1e46